src/encoded/static/components/dataset.js

```diff
@@ -2201,6 +2201,67 @@ const functionalCharacterizationSeriesTableColumns = {
     },
 };
 
+
+/**
+ * Generate a comma-separated string of pulse-chase time durations from the given experiment's biosamples.
+ * @param {object} experiment - The experiment to generate the pulse-chase durations from.
+ * @returns {string} - A comma-separated string of pulse-chase durations.
+ */
+const computePulseDurations = (dataset) => {
+    const biosamples = collectDatasetBiosamples(dataset);
+    let pulseDurations = '';
+    if (biosamples.length > 0) {
+        biosamples.forEach((biosample) => {
+            if (biosample.pulse_chase_time) {
+                pulseDurations = `${biosample.pulse_chase_time} ${biosample.pulse_chase_time_units}${biosample.pulse_chase_time > 1 ? 's' : ''}`;
+            }
+        });
+    }
+    pulseDurations = _.uniq(pulseDurations);
+    return pulseDurations;
+};
```

Comment:
I think the value in keeping it this way is to make sure uniqueness checks all combinations of pulse_chase_time **and** pulse_chase_time_units (so that 2 minutes and 2 hours don't have the '2' removed?)
But there might be a way to do it by getting the values as tuples and then checking for uniqueness before making that a string? If we're not worried about efficiency, I'm happy leaving it as-is

Reply:
Yes, you’re right; I didn’t think about that.